packages/tocco-ui-showcase/src/components/ShowCaseApp.js

```diff
@@ -1,29 +1,53 @@
 import React from 'react'
+import {addLocaleData, IntlProvider} from 'react-intl'
 
 import Navigation from './Navigation'
+import LocaleSwitcher from './LocaleSwitcher'
 import ShowCaseList from './ShowCaseList'
 import Affix from './Affix'
 
+import de from 'react-intl/locale-data/de'
+import en from 'react-intl/locale-data/en'
+import fr from 'react-intl/locale-data/fr'
+import it from 'react-intl/locale-data/it'
+
 export default class ShowCaseApp extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      locale: 'en'
+    }
+
+    addLocaleData([...de, ...en, ...fr, ...it])
+  }
+
   static propTypes = {
     componentsTree: React.PropTypes.array.isRequired
   }
 
+  localeChange(locale) {
+    console.log('Change locale to: ', locale)
```

Comment:
its only showcase app code. i left that there on purpose so you see the exact locale name in console.